mongofiles/mongofiles.go

```diff
@@ -111,7 +115,19 @@ func (mf *MongoFiles) ValidateCommand(args []string) error {
 		} else {
 			mf.FileName = args[1]
 		}
-	case Search, Put, Get, Delete:
+	case Put, Get:
+		// monogofiles --put and mongofiles --get should work over
+		// a list of files, i.e. by using mf.FileNameList -- see
+		// TOOLS-2667 for user-level specification
```

Comment:
We don't usually refer to tickets in the code comment. Apart from that, this comment is confusing; what do you mean by "user-level specification"? [TOOLS-2667](https://jira.mongodb.org/browse/TOOLS-2667) has no description or other details...

Reply:
I meant "user specification" as in "how _mongofiles_ should be called from the command-line" -- maybe the terminology was misused there. In any case, I can change this comment to be more clear and to not refer to a specific ticket.